<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rpm_widget</class>
 <widget class="QWidget" name="rpm_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: none;</string>
  </property>
  <widget class="QLabel" name="rpmCaption">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>238</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Levenim MT</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white; background:none;</string>
   </property>
   <property name="text">
    <string>RPM * 1000/min</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent; </string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="rpmNumber">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>173</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: yellow;
background-color: rgba(255, 255, 255, 20)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="digitCount">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="black_circle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>301</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: none;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img/blackcircle.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="dots_widget" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>301</height>
    </rect>
   </property>
   <widget class="QLabel" name="circle1">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>195</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/colorized_circles/circle1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="circle2">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>172</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/colorized_circles/circle2.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="circle3">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>148</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/colorized_circles/circle3.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="circle4">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>124</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/colorized_circles/circle4.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="circle5">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>99</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/colorized_circles/circle5.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="circle6">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>75</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/colorized_circles/circle6.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="circle7">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>54</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/colorized_circles/circle7.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="circle8">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>36</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/colorized_circles/circle8.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="circle9">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>21</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/colorized_circles/circle9.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="circle10">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>11</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/colorized_circles/circle10.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="circle11">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>4</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/colorized_circles/circle11.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="circle12">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>1</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/colorized_circles/circle12.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="circle13">
    <property name="geometry">
     <rect>
      <x>159</x>
      <y>5</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/colorized_circles/circle13.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="circle14">
    <property name="geometry">
     <rect>
      <x>182</x>
      <y>11</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/colorized_circles/circle14.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="circle15">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>21</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/colorized_circles/circle15.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="circle16">
    <property name="geometry">
     <rect>
      <x>224</x>
      <y>37</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/colorized_circles/circle16.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="circle17">
    <property name="geometry">
     <rect>
      <x>241</x>
      <y>55</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/colorized_circles/circle17.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="circle18">
    <property name="geometry">
     <rect>
      <x>253</x>
      <y>75</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/colorized_circles/circle18.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="circle19">
    <property name="geometry">
     <rect>
      <x>263</x>
      <y>98</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/colorized_circles/circle19.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="circle20">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>121</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/colorized_circles/circle20.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="circle21">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>145</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/colorized_circles/circle21.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="circle22">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>169</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/colorized_circles/circle22.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="circle23">
    <property name="geometry">
     <rect>
      <x>257</x>
      <y>191</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/colorized_circles/circle23.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="pintop">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img/pintop.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>black_circle</zorder>
  <zorder>graphicsView</zorder>
  <zorder>dots_widget</zorder>
  <zorder>rpmCaption</zorder>
  <zorder>rpmNumber</zorder>
  <zorder>pintop</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
